otter/worker/launch_server_v1.py

```diff
@@ -308,6 +308,13 @@ def _check_server_created(f):
         d.addBoth(_check_results, f)
         return d
 
+    def _create_with_delay():
+        d = _treq.post(path, headers=headers(auth_token),
+                       data=json.dumps({'server': server_config}), log=log)
+        # Add 1 second delay to space 1 second between server creations
+        d.addCallback(delay, clock, 1)
```

Comment:
It is a bit unfortunate that this gets the result, and then sits on it for a second. We need to space outgoing requests, but you can still asynchronously space results without having to wait on them here.